nengo/conftest.py

```diff
@@ -18,8 +19,7 @@
 
 
 def pytest_configure(config):
-    rc.reload_rc([])
-    rc.set('decoder_cache', 'enabled', 'False')
+    matplotlib.use('agg')
```

Comment:
I've never tried the 'pdf' backend, so I'd be wary of changing to it right before a release unless we have a good reason. But if someone wants to do some more research, this could be a good option down the road. I'm not sure what the pros and cons would be.